lapixdl/convert.py

```diff
@@ -0,0 +1,251 @@
+
+    if df_out.empty:
+        raise ValueError('Data without valid geometries! After transform the geometries the dataframe stay empty.')
+
+    return df_out
+
+
+def from_labelbox(
+    labelbox_df: pd.DataFrame,
+    schematic_to_id: dict[str, int]
+) -> Lapix:
+    '''Transform the raw dataframe from LabelBox data to Lapix dataframe'''
+
+    df_out = labelbox_df.copy()
+
+    # Drop ignored images at labelling process
+    if 'Skipped' in df_out.columns:
+        df_out = df_out.drop(df_out[df_out['Skipped']].index)
+
+    # Drop irrelevant columns
+    df_out = df_out.drop(
+        [
+            'DataRow ID', 'Labeled Data', 'Created By', 'Project Name', 'Dataset Name', 'Created At', 'Updated At',
+            'Seconds to Label', 'Agreement', 'Benchmark Agreement', 'Benchmark ID', 'View Label',
+            'Has Open Issues', 'Skipped',
+        ], axis=1, errors='ignore',
+    )
+
+    # Get image names
+    df_out['image_name'] = df_out.apply(lambda row: basename(row['External ID']), axis=1)
+    df_out = df_out.drop(['External ID'], axis=1)
+
+    # Remove duplicated labels
+    df_out = __lbox_drop_duplicate_labels(df_out)
+
+    # Explode annotations to each row
+    df_out = __lbox_explode_images_annotations(df_out)
+
+    # Transform labelbox annotation to a geometry
+    df_out = __lbox_cast_geometries(df_out)
+
+    # Map category IDs
+    df_out['category_id'] = df_out.apply(lambda row: schematic_to_id[row['objects']['schemaId']], axis=1)
+
+    df_out = df_out.drop(['ID', 'objects', 'Reviews'], axis=1)
+
+    lapix_df = Lapix(df_out)
+
+    return lapix_df
+
+
+# -----------------------------------------------------------------------
+# Functions work with data to COCO
+def bounds_to_coco_bb(
+    bounds: tuple[float],
+    decimals: int = 2,
+) -> list[float]:
+    # bounds is in  (minx, miny, maxx, maxy)
+    # bb  of coco is in [min(x), min(y), max(x)-min(x), max(y)-min(y)]
+    b = tuple(np.round(x, decimals) for x in bounds)
+    min_x, min_y, max_x, max_y = b
+    return [min_x, min_y, max_x - min_x, max_y - min_y]
+
+
+def pol_to_coco_segment(
+    geo: Polygon | MultiPolygon,
+    decimals: int = 2,
+) -> list[list[float]]:
+    # polygon of shapely is a class
+    # polygon or segmentation at coco is a list of [[x0, y0, x1, y1 ...]]
+
+    def coco_pol(geometry: Polygon, decimals: int = decimals) -> list[float]:
+        list_of_points = list(zip(*geometry.exterior.coords.xy))
+        geometry = list(chain(*list_of_points))
+        geometry = list(np.around(np.array(geometry), decimals))
+        return geometry
+
+    if geo.geom_type == 'Polygon':
+        return [coco_pol(geo)]
+    elif geo.geom_type == 'MultiPolygon':
+        return [coco_pol(g) for g in geo.geoms]
+    else:
+        raise TypeError(f'Geometry shape is not a polygon or MultiPolygon. This is a {geo.geom_type}.')
+
+
+def __to_OD_COCO_annotations(
```

Comment:
```suggestion
def __to_od_coco_annotations(
```
the pattern for method names in this project seems to be `lower_case`

Reply:
Done on [3e30be4](https://github.com/lapix-ufsc/lapixdl/pull/34/commits/3e30be492800341fe79636e124089e2425f45d21)